brands/ghostery/branding/pref/firefox-branding.js

```diff
@@ -17,157 +19,116 @@ pref("app.update.url.manual", "https://www.ghostery.com/download/");
 // A default value for the "More information about this update" link
 // supplied in the "An update is available" page of the update wizard.
 pref("app.update.url.details", "https://www.ghostery.com/%LOCALE%/browser/notes");
-
-// Disable System Addon updates
-pref("extensions.systemAddon.update.url", "");
-
 // The number of days a binary is permitted to be old
 // without checking for an update.  This assumes that
 // app.update.checkInstallTime is true.
 pref("app.update.checkInstallTime.days", 63);
-
 // Give the user x seconds to reboot before showing a badge on the hamburger
 // button. default=immediately
 pref("app.update.badgeWaitTime", 0);
 
-// Number of usages of the web console.
-// If this is less than 5, then pasting code into the web console is disabled
-pref("devtools.selfxss.count", 5);
+/** VARIOUS ***/
+// Fetch shavar updates from Ghostery endpoint
+pref("browser.safebrowsing.provider.mozilla.updateURL", "https://get.ghosterybrowser.com/shavar/downloads?client=SAFEBROWSING_ID&pver=2.2");
+
+// UA compat mode - Adds Firefox/VER to the UA string in addition to the APP_NAME. (https://github.com/ghostery/user-agent-desktop/issues/114)
+pref("general.useragent.compatMode.firefox", true);
+
+// Support and feedback URLs
+pref("app.support.baseURL", "https://get.ghosterybrowser.com/support/");
+pref("app.feedback.baseURL", "https://www.ghostery.com/support/");
+
+// Override settings server to Ghostery
+pref("services.settings.server", "https://get.ghosterybrowser.com/settings/v1");
+
 
-/** Anti-tracking settings */
-// tracker storage partitioning - currently undocumented setting to partition browser storage for trackers in 3rd party contexts.
-// See https://bugzilla.mozilla.org/show_bug.cgi?id=1549587
-pref("network.cookie.cookieBehavior", 5);
-// origin trimming - controls how much referrer to send across origins, 2 = only send the origin
-// https://wiki.mozilla.org/Security/Referrer
-pref("network.http.referer.XOriginTrimmingPolicy", 2);
-// samesite cookies - lax by default. Protects against CSRF attacks
-// https://hacks.mozilla.org/2020/08/changes-to-samesite-cookie-behavior/
-pref("network.cookie.sameSite.laxByDefault", true);
```

Comment:
why this is removed? I don't see it covered by Better-Fox file #484

Reply:
`XOriginTrimming` is in the BetterFox file [here](https://github.com/ghostery/user-agent-desktop/blob/19706e3396cf9a3eea8255ef98409bf4602eeddf/brands/ghostery/branding/pref/Better-Fox.js#L142-L143).

`sameSite` was removed because SameSite cookie prefs are still considered experimental in Firefox Nightly. As a principle, I excluded Nightly experimental prefs.

Do you want to include sameSite prefs in Betterfox?
```
pref("network.cookie.sameSite.laxByDefault", true);
pref("network.cookie.sameSite.noneRequiresSecure", true);
pref("network.cookie.sameSite.schemeful", false);
```

If so, do we want `schemeful` set to `true`? Are we concerned with breakage with that specifically?

Reply:
please keep the current setting. We have tested it together with @sammacbeth and can confirm they are good for Ghostery browser. 

Let's not introduce any more changes right now, `schemeful` can got in another round if you think it's worth trying.

Reply:
> please keep the current setting. We have tested it together with sammacbeth and can confirm they are good for Ghostery browser.

`pref(network.http.referer.XOriginPolicy", 0)` This is the same as Firefox, so I will remove this pref in `Betterfox.js` since we are discarding any prefs that = Firefox defaults.

`pref("network.http.referer.XOriginTrimmingPolicy", 2);` As mentioned previously, this is included in `Better-fox.js` and will be retained.


> Let's not introduce any more changes right now, `schemeful` can got in another round if you think it's worth trying.

I will add add `pref("network.cookie.sameSite.laxByDefault", true);` and `pref("network.cookie.sameSite.noneRequiresSecure", true);` to `Better-Fox.js`, so lookout for that.

I am still concerned about breakage with `schemeful`. I can include it into Better-fox but leave it commented out since it is something we would like to enable in the future. Is that okay with you, @chrmod ?

Reply:
Yes, that sounds good.